hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -103,4 +103,13 @@ void stop() throws Exception {
         .getInstance(OzoneManagerServiceProvider.class);
     ozoneManagerServiceProvider.stop();
   }
+
+  OzoneManagerServiceProvider getOzoneManagerServiceProvider() {
```

Comment:
If using DI properly this should never be needed, unless used in non-DI code path

Reply:
Yes, I agree. I had created them for readability. I can move it to private.  We don't expect external consumers for these other than tests maybe.

Reply:
Done.